common-api/src/main/java/org/pmiops/workbench/cdr/dao/DomainInfoDao.java

```diff
@@ -42,16 +42,16 @@
    */
   @Query(
       value =
-          "select new org.pmiops.workbench.cdr.model.DomainInfo(\n"
+          "select new org.pmiops.workbench.cdr.model.DbDomainInfo(\n"
               + "d.domain, d.domainId, d.name, d.description,\n"
               + "d.conceptId, COUNT(*), 0L, d.participantCount)\n"
-              + "from DomainInfo d\n"
-              + "join Concept c ON d.domainId = c.domainId\n"
+              + "from DbDomainInfo d\n"
```

Comment:
I remember when I went through this I had to manually fix up one spot where there was a query. But that may only be true because it happened to be a legacy SQL query.

Reply:
I had to manually change DbConcept below. For some reason Intellij did not refactor Concept to DbConcept in this specific query string when using it's refactor functionality(Shift+F6).

Reply:
I think it treats it like a regular string literal. I.e. it doesn't take into consideration that it's supposed to be a query, or that the class name maps to a table name. You have to be pretty careful with the checkboxes when renaming, since there are some imperfect heuristics or logic going on.